config/environments/production.rb

```diff
@@ -102,6 +102,7 @@
     :authentication => ENV['SMTP_AUTH_METHOD'] || :plain,
     :openssl_verify_mode => ENV['SMTP_OPENSSL_VERIFY_MODE'] || 'peer',
     :enable_starttls_auto => ENV['SMTP_ENABLE_STARTTLS_AUTO'] || true,
+    :ca_file => ENV['SMTP_SSL_CERT_FILE'] || "/etc/ssl/certs/ca-certificates.crt",
```

Comment:
I'm not sure this latter value is a standard location we can rely on.

This might be a fix for some servers ... but I think a better improvement here would be to improve the documentation to show how ruby can be set up with correct CA certs, instead of having  what is basically a work-around as the standard approach.